pkg/scaffold/project/gopkg.go

```diff
@@ -327,6 +327,11 @@ version="kubernetes-1.10.0"
 name="k8s.io/client-go"
 version="kubernetes-1.10.1"
 
+# Required version for client-go 1.10
+[[override]]
+name = "github.com/Azure/go-autorest"
+version = "9.1.0"
+
```

Comment:
We will soon be migrating to k8s 1.11 (coming Tuesday). Will this override go away with that ?